tests/lib/rules/no-implicit-globals.js

```diff
@@ -0,0 +1,233 @@
+            errors: [
+                {
+                    message: "Implicit global variable, assign as window property instead.",
+                    type: "VariableDeclarator"
+                },
+                {
+                    message: "Implicit global variable, assign as window property instead.",
+                    type: "VariableDeclarator"
+                }
+            ]
+        },
+        {
+            code: "let foo = 1;",
+            ecmaFeatures: { blockBindings: true },
+            errors: [
+                {
+                    message: "Implicit global variable, assign as window property instead.",
+                    type: "VariableDeclarator"
+                }
+            ]
+        },
+        {
+            code: "const foo = 1;",
+            ecmaFeatures: { blockBindings: true },
+            errors: [
+                {
+                    message: "Implicit global variable, assign as window property instead.",
+                    type: "VariableDeclarator"
+                }
+            ]
+        },
+        {
+            code: "function foo() {}",
+            errors: [
+                {
+                    message: "Implicit global variable, assign as window property instead.",
+                    type: "FunctionDeclaration"
+                }
+            ]
+        },
+        {
+            code: "foo = function() {};",
+            errors: [
+                {
+                    message: "Implicit global variable, assign as window property instead.",
+                    type: "AssignmentExpression"
+                }
+            ]
+        },
+        {
+            code: "var foo = function() {};",
+            errors: [
+                {
+                    message: "Implicit global variable, assign as window property instead.",
+                    type: "VariableDeclarator"
+                }
+            ]
+        },
+        {
+            code: "let foo = function() {};",
+            ecmaFeatures: { blockBindings: true },
+            errors: [
+                {
+                    message: "Implicit global variable, assign as window property instead.",
+                    type: "VariableDeclarator"
+                }
+            ]
+        },
+        {
+            code: "const foo = function() {};",
+            ecmaFeatures: { blockBindings: true },
+            errors: [
+                {
+                    message: "Implicit global variable, assign as window property instead.",
+                    type: "VariableDeclarator"
+                }
+            ]
+        },
+        {
+            code: "var foo = function foo() {};",
+            errors: [
+                {
+                    message: "Implicit global variable, assign as window property instead.",
+                    type: "VariableDeclarator"
+                }
+            ]
+        },
+        {
+            code: "foo = 1;",
+            ecmaFeatures: { globalReturn: true },
```

Comment:
So it appears in the global return, but not module case, an implicit global will still be created. Sound right?